### DISCUSSION TITLE
How to access result of the query in the scaled job/deployment?

### DISCUSSION THREAD

**Discussion by gitprady** [kedacore/keda#2197] [Q&A / Need Help]
Keda is spinning up pods based on the number of documents as result of query specified in triggers metadata.

Example : 
triggers:
  - type: mongodb
    metadata:
      dbName: "test"
      collection: "test_collection"
      query: '{"type":"book","state":"waiting"}'
      queryValue: "1"
      metricName: "global-metric"

Scenario:
Mongodb has 2 records of such type hence keda creates two pods in K8.

Question : How do I consume this document in the pod. to update/process  this specific document.

Many thanks for your help in advance.

**Comment by zroubalik** [comment]
Hi, I am not sure I fully understand your question. But KEDA doesn't manipulate the data, it is just doing the scaling related actions. Workload/data handling is something that have to be managed by your workload (ie. the pod that you are scaling).
